modules/darwin/src/main/ts/ephox/darwin/queries/CellOpSelection.ts

```diff
@@ -0,0 +1,40 @@
+import { Arr, Fun, Optional } from '@ephox/katamari';
+import { Attribute, SugarElement } from '@ephox/sugar';
+import { Structs, RunOperation } from '@ephox/snooker';
+import { Selections } from '../selection/Selections';
```

Comment:
These imports aren't in order... I wonder why the linter isn't complaining about that :thinking: It should be:
```
import { Ephemera } from '../api/Ephemera';
import * as TableSelection from '../api/TableSelection';
import { Selections } from '../selection/Selections';
import * as SelectionTypes from '../selection/SelectionTypes';
```